pages/first-principles.markdown

```diff
@@ -229,6 +253,8 @@ In complicated/enterprise environments, it's very common for a project to requir
 
 Invariably, Team A's work will require some amount of output from Team B.
 
+:speech_balloon: I think that the important thing here is to minimize cross-team dependencies, not serialized development. I would actually argue that on a personal level, most developers would prefer serializing their work instead of context-switching between multiple threads.
```

Comment:
Agreed, wasn't trying to insinuate developers context-switch; if anything parallel development would reduce that IMO. This section is not complete; will flush out with something more concrete.